src/Translations.js

```diff
@@ -74,37 +76,58 @@ function Translations({ document: { displayed: document, draft, published } }) {
   })
 
   return (
-    <article className={styles.component}>
-      <h3>Vertalingen</h3>
-      {isLoading && <p>Laden...</p>}
-      {isError && <p>Er ging iets mis...</p>}
-
-      {isSuccess && (
-        (published || draft)
-          ? <Languages
-              original={document}
-              {...{ translations, schemaType }}
-              onTranslateFresh={language => {
-                addFreshTranslation({ original: document, language })
-              }}
-              onTranslateDuplicate={language => {
-                addDuplicateTranslation({ original: document, language })
-              }}
-            />
-          : <p>Het lijkt erop dat er nog niets is om te vertalen!</p>
-      )}
+    <Container
+      paddingBottom={9}
+      paddingTop={5}
+      paddingX={4}
+      sizing='border'
+      width={1}
+    >
+      <Stack space={2}>
+        <Text weight='semibold'>Vertalingen</Text>
+        
+        {isLoading && (
+          <Flex justify="center">
+            <Spinner muted />
+          </Flex>
+        )}
+        
+        {isError && (
+          <Card padding={[3, 3, 4]}
+            radius={2}
+            shadow={1}
+            tone='critical'
+          >
+            <Text>Er ging iets mis...</Text>
```

Comment:
Hadden we niet gezegd dat plugins standaard in het Engels zijn? Ik weet het niet meer.

Reply:
Ik weet het ook niet meer precies, maar volgens mij hebben we dat inderdaad bij de SEO plugin doorgevoerd.

Reply:
Die analyse is nu in het Nederlands: 
<img width="679" alt="image" src="https://user-images.githubusercontent.com/4712314/158400464-a7df3009-d4e6-4d9b-a336-06f2b8544339.png">

Maar daar hoeven we in principe alleen de vertalingen uit te halen, default is nl. Engels. Zullen we dan Engels aanhouden? Dan kan ik de plugins wel even af.